Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -590,7 +593,7 @@ def get_azure_cloud(params, integration_name):
                                              })
 
     # There is no need for backward compatibility support, as the integration didn't support it to begin with.
-    return AZURE_CLOUDS[AzureCloudNames.WORLDWIDE]
+    return AZURE_CLOUDS[azure_cloud_arg]
```

Comment:
  > return AZURE_CLOUDS.get(azure_cloud_arg, AZURE_WORLDWIDE_CLOUD)
  in case like in `AzureKeyVault` the `azure_cloud_arg` can be '' without `server_url` or  `azure_ad_endpoint` in params